### PR TITLE
Fix the MacOS resource generation

### PR DESCRIPTION
**Pull request by bakulf**
This script is now broken because of the macos menu localization

### CODE REVIEW

File: scripts/import_macos_pkg_resources.py [hunk 1]
```diff
@@ -91,16 +91,16 @@ def translate(root, locale):
...
 for translation in os.listdir('translations'):
-    translationFile = os.path.join('translations', translation)
-    if not os.path.isfile(translationFile):
+    translationDir = os.path.join('translations', translation)
+    if not os.path.isdir(translationDir):
         continue
 
-    if not translation.startswith('mozillavpn_') or not translation.endswith('.ts'):
+    translationFile = os.path.join(translationDir, f"mozillavpn_{translation}.ts")
+    print(translationFile)
```
Comment (flodolo) [inline scripts/import_macos_pkg_resources.py:99]: Is this print a left over?

File: .github/l10n/check_l10n_issues.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
     exceptions = json.load(f)
 
 # Load the .ts file
-ts_file = os.path.join(vpn_root_folder, "translations", "mozillavpn_en.ts")
+ts_file = os.path.join(vpn_root_folder, "translations", "en", "mozillavpn_en.ts")
```
Comment (flodolo) [inline .github/l10n/check_l10n_issues.py:24]: Is it expected that the linter is now failing? Would it fix itself once merged?

`Updating 'translations/mozillavpn_en.ts'`, which means the file in `en` doesn't exist.

Comment (flodolo) [reply .github/l10n/check_l10n_issues.py:24]: I think this line should be reverted. This script extracts its own .ts file in `translations/mozillavpn_en.ts`. As far as I understand, the import script relies on `translations.pri` for extracting them in subfolders, and that's not needed here.

Comment (bakulf) [reply .github/l10n/check_l10n_issues.py:24]: Right. This file now is into an "en" folder.

**Review by flodolo — APPROVED** [review]

**Pull request merged**